### ISSUE TITLE
Missing normalization with 0.3.0 and 0.4.0 models

### ISSUE THREAD

**Issue by iimog** [chfc-cmi/cmr-seg-tl#2]
In the pkl files for Model version 0.3.0 and 0.4.0 the normalization step with `imagenet_stats` was lost. This causes segmentations to fail:

<img width="1224" height="407" alt="Image" src="https://github.com/user-attachments/assets/b259671b-0d0e-46c9-9739-62cd9c011c1d" />

The `state_dict`s of the models is indeed identical. However, the `after_batch` transformation of the dataloaders is missing the Normalize step:

```
trainedModel_020.dl.after_batch
Pipeline: IntToFloatTensor -- {'div': 255.0, 'div_mask': 1} -> Flip -- {'size': None, 'mode': 'bilinear', 'pad_mode': 'reflection', 'mode_mask': 'nearest', 'align_corners': True, 'p': 0.5} -> Brightness -- {'max_lighting': np.float64(0.4), 'p': 1.0, 'draw': None, 'batch': False} -> Normalize -- {'mean': tensor([[[[0.4850]], [[0.4560]], [[0.4060]]]]), 'std': tensor([[[[0.2290]], [[0.2240]], [[0.2250]]]]), 'axes': (0, 2, 3)}
```
vs
```
trainedModel_030.dl.after_batch
Pipeline: IntToFloatTensor -- {'div': 255.0, 'div_mask': 1} -> Flip -- {'size': None, 'mode': 'bilinear', 'pad_mode': 'reflection', 'mode_mask': 'nearest', 'align_corners': True, 'p': 0.5} -> Brightness -- {'max_lighting': np.float64(0.4), 'p': 1.0, 'draw': None, 'batch': False}
```

Transplanting the after_batch pipeline, makes the model work again:
```
trainedModel_030.dl.after_batch = trainedModel_020.dl.after_batch
```
